### DISCUSSION TITLE
How to search in the files part of cmake build but outside workspace folder?

### DISCUSSION THREAD

**Discussion by MaJerle** [microsoft/vscode-cmake-tools#2826] [General]
If I have external libraries and want to search in the vscode with CTRL+F, how can I include the files outside the workspace folder?
Files I'd like to search in, are part of CMake build tho

Is there a way to do so?

**Comment by bobbrow** [comment]
I'm not aware of a way to do this today.  Our extension doesn't have any interactions with Search.  I would recommend opening an issue with the VS Code team.  https://github.com/microsoft/vscode/issues/new/choose

**Comment by mike-jn** [comment]
I have the same issue and couldn't find a solution. @MaJerle did you end up solving this?

**Reply by MaJerle** [reply]
I did not. Need explorer for that.

**Reply by mike-jn** [reply]
I ended up creating a multi-root workspace and just added the other projects as additional folders.
